### PR TITLE
feat(chats): Typing indicator

### LINKED ISSUE #336: feat(Chats): Start wiring up typing indicators in chat
**Task:**
Like the title says, we want to add the typing indicator into UplinkWeb. We had this in Core so maybe there will be some stuff we could use from there? 

Typing indicator should appear in the chat and on the sidebar

### PR DESCRIPTION
**Pull request by Flemmli97**
### What this PR does 📖

- Implements the typing indicator fully for chats
  - Icon will change to indicate someone is typing
  - Below the chatbar a message will appear with the people who are typing

### Which issue(s) this PR fixes 🔨

- Resolve #336

### Note

When typing an event is send to warp via wasm. That process logs an error with something along the line of `recursive function` etc but this error does not impact the functionality.
We can try to resolve this at a later time though. Considering (while this case happens all the time here and thus is reproducable) it happens quite a lot at other times too so they could all have the same root cause.

### CODE REVIEW

**Review by InfamousVague — APPROVED** [review]

File: src/lib/components/Polling.svelte [hunk 1]
```diff
@@ -18,6 +19,10 @@
         setTimeout(poll, currentInterval)
     }
 
+    async function updateTypingIndicators() {
+        UIStore.updateTypingIndicators()
```
Comment (stavares843) [inline src/lib/components/Polling.svelte:23]: ReferenceError: UIStore is not defined


Captura de ecrã 2024-08-07, às 12 56 16

**Comment by stavares843** [comment]
https://github.com/user-attachments/assets/2231417a-e03b-4927-9a86-bc77f17d717f

when you type and clear out, keeps saying you are typing even after more than 1 minute

**Comment by stavares843** [comment]
Captura de ecrã 2024-08-07, às 12 57 24


the typing indicator could have a little bit more of padding below the input field

**Comment by stavares843** [comment]
all the above was on chrome and firefox in macos

**Comment by stavares843** [comment]
Is this happening on your side? @Flemmli97 
https://github.com/Satellite-im/UplinkWeb/pull/364#issuecomment-2273300611

**Comment by Flemmli97** [comment]
> Is this happening on your side? @Flemmli97 [#364 (comment)](https://github.com/Satellite-im/UplinkWeb/pull/364#issuecomment-2273300611)

is it still happening? it shouldve been fixed as the ReferenceError is the part updating that stuff

**Comment by stavares843** [comment]
thank you :tada:

**Pull request merged**